src/app/containers/Header/index.jsx

```diff
@@ -32,6 +32,8 @@ const HeaderContainer = () => {
 
   const isOperaMini = useOperaMiniDetection();
 
+  const brandFocusRef = useRef(null);
```

Comment:
As this is no longer a reference to the element which can be focussed, perhaps it should become simply brandRef?